src/plugins/intel_gpu/src/graph/graph_optimizer/prepare_primitive_fusing.cpp

```diff
@@ -1063,11 +1063,6 @@ void prepare_primitive_fusing::fuse_simple_primitives(program &p) {
                 auto eltw_in_size = peer_node->get_output_layout();
                 if (eltw_in_size.is_dynamic())
                     return;
-                // When input rank > 4, fused eltwise to gemm should be converted to 4 dim in init_onednn_primitive_attribute()
-                // But current init_onednn_primitive_attribute() cannot handle dynamic shape case.
-                auto eltw_in_rank = fused_node->get_output_layout().get_rank();
-                if ((fused_node->is_type<gemm>()) && (eltw_in_rank > 4))
-                    return;
             }
```

Comment:
Isn't this needed any more for the original purpose too?

Reply:
No. My approach about this case is little bit far from existing unit test of gemm-eltwise fusing with dynamic shape.
So actual gemm-eltwise fusing case doesn't make get_tensor() issue in add_onednn_optimization_attributes pass.